sympy/physics/control/tests/test_lti.py

```diff
@@ -322,3 +322,152 @@ def test_TransferFunction_is_biproper():
+    assert Series(tf1, tf2, evaluate=True) == Series(tf1, tf2).doit()
+    assert Series(tf1, tf2, Parallel(tf1, -tf3), evaluate=True) == \
+        Series(tf1, tf2, Parallel(tf1, -tf3)).doit()
+    assert Series(tf2, tf1, -tf3, evaluate=True) == Series(tf2, tf1, -tf3).doit()
+    assert not Series(tf1, -tf2, evaluate=False) == Series(tf1, -tf2).doit()
+
+    assert Series(Parallel(tf1, tf2), Parallel(tf2, -tf3)).doit() == \
+        TransferFunction((k*(omega_n**2 + 2*omega_n*s*zeta + s**2) + 1)*(-a2*p + k*(a2*s + p) + s),\
+            (a2*s + p)*(omega_n**2 + 2*omega_n*s*zeta + s**2), s)
+    assert Series(-tf1, -tf2, -tf3).doit() == \
+        TransferFunction(k*(-a2*p + s), (a2*s + p)*(omega_n**2 + 2*omega_n*s*zeta + s**2), s)
+    assert Series(tf2, tf3, Series(Parallel(tf2, -tf1), tf3)).doit() == \
+        TransferFunction(k*(a2*p - s)**2*(k*(omega_n**2 + 2*omega_n*s*zeta + s**2) - 1),\
+            (a2*s + p)**2*(omega_n**2 + 2*omega_n*s*zeta + s**2), s)
+
+
+def test_Parallel_construction():
+    zeta, wn = symbols('zeta, wn')
+    tf = TransferFunction(a0*s**3 + a1*s**2 - a2*s, b0*p**4 + b1*p**3 - b2*s*p, s)
+    tf2 = TransferFunction(a2*p - s, a2*s + p, s)
+    tf3 = TransferFunction(a0*p + p**a1 - s, p, p)
+    tf4 = TransferFunction(1, s**2 + 2*zeta*wn*s + wn**2, s)
+    inp = Function('X_d')(s)
+    out = Function('X')(s)
+
+    p0 = Parallel(tf, tf2)
+    assert p0.args == (tf, tf2)
+
+    p1 = Parallel(Series(tf, -tf2), tf2)
+    assert p1.args == (Series(tf, -tf2), tf2)
+
+    p2 = Parallel(tf, Series(inp, -out), tf2)
+    assert p2.args == (tf, Series(X_d, -X), tf2)
+
+    p3 = Parallel(tf, tf2, tf4)
+    assert p3.args == (tf, tf2, tf4)
+
+    p4 = Parallel(inp, out)
+    assert p4.args == (X_d, X)
+
+    p5 = Parallel(tf, tf2)
+    assert p0 == p5
+    assert not p0 == p1
+
+    p6 = Parallel(tf2, tf4, Series(tf2, -tf4))
+    assert p6.args == (tf2, tf4, Series(tf2, -tf4))
+
+    p7 = Parallel(tf2, tf4, Parallel(Series(tf2, -tf), tf4))
+    assert p7.args == (tf2, tf4, Parallel(Series(tf2, -tf), tf4))
+
+    raises(ValueError, lambda: Parallel(tf, tf3))
+    raises(ValueError, lambda: Parallel(tf, tf2, tf3, tf4))
+    raises(ValueError, lambda: Parallel(-tf3, tf4))
+    raises(ValueError, lambda: Parallel(2, tf, tf4))
+    raises(ValueError, lambda: Parallel(s**2 + p*s, tf3, tf2))
+    raises(ValueError, lambda: Parallel(tf3, Matrix([1, 2, 3, 4])))
+
+
+def test_Parallel_functions():
+    zeta, omega_n, wn = symbols('zeta, omega_n, wn')
+    tf1 = TransferFunction(1, s**2 + 2*zeta*wn*s + wn**2, s)
+    tf2 = TransferFunction(k, 1, s)
+    tf3 = TransferFunction(a2*p - s, a2*s + p, s)
+    tf4 = TransferFunction(a0*p + p**a1 - s, p, p)
+
+    assert tf1 + tf2 + tf3 == Parallel(tf1, tf2, tf3)
+    assert tf1 + tf2*tf3 == Parallel(tf1, Series(tf2, tf3))
+    assert tf2 + tf3 + tf2*tf1 - tf3 == Parallel(tf2, tf3, Series(tf2, tf1), -tf3)
+    assert -(tf1 + tf2) == -tf1 - tf2 == Parallel(-tf1, -tf2)
```

Comment:
What about `- Parallel(tf1, tf2)` ?

Reply:
Then, I think other operators `+, -, *, /` will also have to be overloaded, for doing something like:
 `>>> Series(tf1, tf2) + Series(tf3, tf4)`

Reply:
> What about `- Parallel(tf1, tf2)` ?

I think the negation of a `Series` or `Parallel` object is not needed.
```python
-tf1*tf2  --> Series(-tf1, tf2)
-tf1*-tf2 --> Series(-tf1, -tf2)
tf1 - tf2 --> Parallel(tf1, -tf2)
-tf1 - tf2 --> Parallel(-tf1, -tf2)
```
It is my fault, `-(tf1 + tf2)` is not equal to `-tf1 - tf2`.

Reply:
Yeah, but, `(tf1 * tf2) + (tf3 * tf4)` would be processed as `Series(tf1, tf2) + Series(tf3, tf4)` and  something like `tf1 + (tf2 * tf3)` as `tf1 - Series(tf2, tf3)` which is how we get `Parallel(tf1, Series(tf2, tf3))`. So, wouldn't this be required anyway?

Reply:
You can begin with `Series(tf1, tf2) + Series(tf3, tf4)` to return `Series(Series(tf1, tf2), Series(tf3, tf4))` and 
`-Parallel(tf1, tf2)` to return `Series(TransferFunction(-1, 1, s), Parallel(tf1, tf2))`

Reply:
> `-Parallel(tf1, tf2)` to return `Series(TransferFunction(-1, 1, s), Parallel(tf1, tf2))`

Yes, I was thinking of the same approach. Also, `tf1*tf2 - tf3*tf4` would return `Parallel(Series(tf1, tf2), -Series(tf3, tf4))`. Since we don't want anything like `Series(tf, Series(...))`, `-Series(tf1, tf2)` would be left as it is.

Reply:
`-Parallel(tf1, tf2)` can return `Series(TransferFunction(-1, 1, s), Parallel(tf1, tf2))`, what about `-Series(tf1, tf2)`?? It can't be `Series(TransferFunction(-1, 1, s), tf1, tf2)` because `-tf1*tf2` and `-(tf1*tf2)` are different. I think we need two Series instances here (previously discussed [here](https://github.com/sympy/sympy/pull/19390#discussion_r442365458)). So it would be `Series(TransferFunction(-1, 1, s), Series(tf1, tf2))`. can anyone comment here?

Reply:
ping @sylee957 @ishanaj @Sc0rpi0n101 here

Reply:
> I think we need two Series instances (previously discussed [here](https://github.com/sympy/sympy/pull/19390#discussion_r442365458)). So it would be `Series(TransferFunction(-1, 1, s), Series(tf1, tf2))`.

If another Series instance is required, we should have it that way.

What I meant to say there was for something like `tf1 * tf2 * tf3` being `Series(tf1, Series(tf2, tf3))`. Here, the inner Series is redundant when Series can support multiple parameters and it can be `Series(tf1, tf2, tf3)`. 

Now, if a nested Series is required to accurately represent something, it should be allowed.